viewing_party/party.py

```diff
@@ -1,23 +1,168 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
+from csv import list_dialects
+
+
+TITLE_KEY = "title"
+GENRE_KEY = "genre"
+RATING_KEY = "rating"
+
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {
+        TITLE_KEY: title,
+        GENRE_KEY: genre,
+        RATING_KEY: rating}
+
+    if not new_movie[TITLE_KEY]:
+        return None
+    if not new_movie[GENRE_KEY]:
+        return None
+    if not new_movie[RATING_KEY]:
+        return None
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+    user_data = {"watched": [movie]}
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data = {"watchlist": [movie]}
+    return user_data
+
+
+def watch_movie(user_data, title):
+    if user_data["watchlist"]:

```

Comment:
I've been doing it this way and I'm happy I came back to read this and go over it because I agree. I didn't think of it like that, but it makes sense to end things quickly if they aren't working out then to keep the program running. I guess I did this because it might have expected a 'return None' if it didn't work out, and the expected data wasn't supplied. But I see your point about killing the program with an error instead of running a check.